src/Orleans.Core/Logging/ILoggerExtensions.cs

```diff
@@ -13,7 +13,9 @@ public static class OrleansLoggerExtension
         /// Writes a log entry at the Debug severity level.
         /// </summary>
         /// <param name="logger">The logger</param>
-        /// <param name="format">A standard format string, suitable for String.Format.</param>
+        /// <param name="format">Format string of the log message with named parameters
+        /// <remarks>Not always suitable for <c>String.Format</c>. See Microsoft.Extensions.Logging usage</remarks>
```

Comment:
can we do: See Microsoft.Extensions.Logging Message Template section. 
just `Microsoft.Extensions.Logging` is too broad, not clear on which documents reader should read

Reply:
Thought the same way, however was not capable to come up with something better. Indeed, the option you proposed is more informative.

Reply:
we can merge this after you update the xml. 
Let's do "Not always suitable for String.Format, See Microsoft.Extensions.Logging MessageTemplate sectionfor more information. Suggest to use their pattern over this extension method"

Since these extension method exists really is just to help runtime migrate to MEL. If runtime uses MEL patterns, we can remove these extension methods.